demisto_sdk/commands/create_artifacts/content_creator.py

```diff
@@ -108,35 +109,57 @@ def run(self):
 
         return 0
 
+    @staticmethod
+    def fix_script_in_unified_yml(yml_content):
+        if 'script' in yml_content:
+            if isinstance(yml_content.get('script'), str):
+                if yml_content.get('script') not in ('-', ''):
+                    yml_content['script'] = FoldedScalarString(yml_content['script'])
+
+            elif yml_content.get('script').get('script') not in ('-', ''):
+                yml_content['script']['script'] = FoldedScalarString(yml_content['script']['script'])
+
     def add_from_version_to_yml(self, file_path=None, yml_content=None, save_yml=True):
         if self.no_fromversion:
             return {}
 
+        ryaml = YAML()
+        ryaml.preserve_quotes = True
+        ryaml.width = 50000  # make sure long lines will not break (relevant for code section)
         if not yml_content:
-            yml_content = get_yaml(file_path)
+            with open(file_path, 'r') as yml_file:
+                yml_content = ryaml.load(yml_file)
 
         if parse_version(yml_content.get('toversion', '99.99.99')) > parse_version(
                 LATEST_SUPPORTED_VERSION) > parse_version(yml_content.get('fromversion', '0.0.0')):
             yml_content['fromversion'] = LATEST_SUPPORTED_VERSION
 
+            if 'detaileddescription' in yml_content:
+                yml_content['detaileddescription'] = FoldedScalarString(yml_content['detaileddescription'])
+
+            self.fix_script_in_unified_yml(yml_content)
+
             if save_yml:
-                with open(file_path, 'w') as f:
-                    yaml.dump(yml_content, f)
+                with open(file_path, mode='w', encoding='utf-8') as f:
+                    ryaml.dump(yml_content, f)
 
         return yml_content
 
     def add_from_version_to_json(self, file_path):
         if self.no_fromversion:
             return {}
 
-        json_content = tools.get_json(file_path)
+        with open(file_path, 'r') as f:
+            json_content = ujson.load(f)
 
         if parse_version(json_content.get('toVersion', '99.99.99')) > parse_version(
                 LATEST_SUPPORTED_VERSION) > parse_version(json_content.get('fromVersion', '0.0.0')):
             json_content['fromVersion'] = LATEST_SUPPORTED_VERSION
 
+            # ujson lets you keep html chars as unicode like "<" should be "\u003c"
             with open(file_path, 'w') as f:
-                json.dump(json_content, f, indent=4)
+                ujson.dump(json_content, f, indent=4, encode_html_chars=True, escape_forward_slashes=False,
```

Comment:
this change is also relevant to format command, please open an issue.

Reply:
done https://github.com/demisto/etc/issues/26423